### PR TITLE
Camelise action name to follow JavaScript conventions

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ export default Ember.Controller.extend({
 
 ### Template
 ```handlebars
-{{#ui-dropdown class="selection" onChange=(action 'update_selected')}}
+{{#ui-dropdown class="selection" onChange=(action 'updateselected')}}
```
Comment (andrewbranch) [inline README.md]: Shouldn’t this be `action 'updateSelected'`? (missing the uppercase “S”)

Comment (locks) [inline README.md]: yup, my bad! I'll fix it later today.

**Comment by locks** [comment]
@andrewbranch done!

**Comment by aaronbhansen** [comment]
thanks

**Pull request merged**